packages/react/src/components/ComboBox/ComboBox.test.tsx

```diff
@@ -794,6 +794,108 @@ describe('ComboBox', () => {
     });
   });
 
+  it('in multiSelect mode, selectAll selects all options', () => {
```

Comment:
What happens if somebody includes a select all option to a ComboBox in single select mode? (Don't know why they would but...)

Reply:
Also if they pass multiple select all options

Reply:
added a test! (the way I made it, it should act like a normal option 😄)